datadict/datadict-oci8.inc.php

```diff
@@ -25,8 +25,15 @@ class ADODB2_oci8 extends ADODB_DataDict {
 	var $alterCol = ' MODIFY ';
 	var $typeX = 'VARCHAR(4000)';
 	var $typeXL = 'CLOB';
+	
+	/*
+	* Legacy compatibility for sequence names for emulated auto-increments
+	* If set to true, creates sequences and triggers as TRIG_394545594
+	* instead of TRIG_possibly_too_long_tablename
+	*/
```

Comment:
- This should be a PHPDoc block (use `/**)
- add a space before the `*` on each subsequent line to align properly.
- Missing period `.`at end of 1st line (*auto-increments*)
- Mention that the number is computed as CRC32 from the table name
- Add `@var bool $useCompactAutoIncrements`